modules/fluid_properties/include/userobjects/SodiumProperty.h

```diff
@@ -0,0 +1,71 @@
+/*************************************************/
+/*           DO NOT MODIFY THIS HEADER           */
+/*                                               */
+/*                     BISON                     */
+/*                                               */
+/*    (c) 2015 Battelle Energy Alliance, LLC     */
+/*            ALL RIGHTS RESERVED                */
+/*                                               */
+/*   Prepared by Battelle Energy Alliance, LLC   */
+/*     Under Contract No. DE-AC07-05ID14517      */
+/*     With the U. S. Department of Energy       */
+/*                                               */
+/*     See COPYRIGHT for full restrictions       */
+/*************************************************/
+
+#ifndef SODIUMPROPERTY_H
+#define SODIUMPROPERTY_H
+
+#include "FluidProperties.h"
+#include "Moose.h"
+
+class SodiumProperty;
```

Comment:
We want to use plural, i.e. `SodiumProperties`, so we are consistent with the rest of the module.